polyglot/piranha/test-resources/strings/input/sample_en.strings

```diff
@@ -0,0 +1,5 @@
+/* Menu item to make the current document plain text */
+"Make Plain Text" = "Make Plain Text";
+
+// Menu item to make the current document rich text
+"Make Rich Text" = "Make Rich Text";
```

Comment:
Why is this file and only this file not ending in newline?

Reply:
Good Catch. I will fix this.